sopel/config/core_section.py

```diff
@@ -198,7 +198,7 @@ def homedir(self):
     host = ValidatedAttribute('host', default='irc.dftba.net')
     """The server to connect to."""
 
-    host_blocks = ListAttribute('host_blocks')
+    host_blocks = ListAttribute('host_blocks', default=[])
```

Comment:
Do we need that `default`?

Reply:
Oh, technically not. `ListAttribute`'s constructor does it if no default is specified: https://github.com/sopel-irc/sopel/blob/185686f92d299f95027f4dccf40ffd994e7807fd/sopel/config/types.py#L317-L318

I'll revert this and the identical change in `nick_blocks` when we reach a consensus on the rST headings and I apply it, to save Travis & LGTM the trouble of running excessive jobs.